src/locale/meson.build

```
@@ -30,5 +31,6 @@ endif
 
 tests += [
         [files('test-localed-util.c',
-               'localed-util.c')],
+               'localed-util.c',
+               'xkbcommon-util.c')],
```

```
FAILED: test-localed-util.p/src_locale_localed-util.c.o 
...
In file included from ../src/src/locale/localed-util.c:27:
../src/src/locale/xkbcommon-util.h:5:10: fatal error: xkbcommon/xkbcommon.h: No such file or directory
    5 | #include <xkbcommon/xkbcommon.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~
```

I guess this now needs an explicit dependency on the xkbcommon headers, similarly to systemd-localed/localectl, as the `xkbcommon/xkbcommon.h` include is now shared with the test as well, e.g.:

```suggestion
               'localed-util.c',
               'xkbcommon-util.c'),
               libxkbcommon.partial_dependency(compile_args: true)],
```

(the dep probably needs to be guarded somehow, see https://github.com/systemd/systemd/blob/ecf4be2952ed13fc1521ae13fcc9543c2f873efe/meson.build#L3044-L3052)

---

Thank you for the suggestion!